docs/Developers/Newcomers_Guide.rst

```diff
@@ -207,6 +209,20 @@ The easy issues that will help you get started are labeled as
 ``difficulty/newcomer`` and are only there to give you a glimpse of what
 it's like to work with us and what the workflow is like.
 
+The coala organisation tries to have ``newcomer`` issues always available
+for people who are joining the organisation.  These newcomer issues are
+intentionally very easy.  The only objective is to give you, the newcomer,
+an opportunity to participate, and learn how to contribute according to
+the coala organisation development guidelines.
+
+coala does **not** follow the GitHub "flow" workflow, so be sure to read
+our guides carefully.  If you do not follow the coala workflow, our team
+will try to nudge you in the right direction.  If nudging isnt effective,
+your contributions will be rejected.  coala will not lower its standards,
```

Comment:
`rejected` sounds ambiguous. People might think they can close "rejected" PR and open a new one, or they might think they can solve another issue.

Reply:
I agree with both ambiguous and aggressive part. Maybe something like "You are expected to comply with the standards in all ways possible. In case you fail to do that, we might assign the issue to someone else who is more appropriate to solve it. In this case, the best you can do is to go through docs and guidelines more thoroughly and abide by them in the future."

Reply:
Instead of rejected we can say `will not be accepted` Same meaning but different way

Reply:
Maybe, "We will be having no option other than rejecting your contributions".